sample-lnd.conf

```diff
@@ -45,29 +53,37 @@
 
 ; Adds an extra domain to the generate certificate. Setting multiple tlsextradomain= entries is allowed.
 ; (old tls files must be deleted if changed)
-; tlsextradomain=
+; Default:
+;   tlsextradomain=
+; Expample (option can be specified multiple times):
+;   tlsextradomain=my-node-domain.com
 
 ; If set, then all certs will automatically be refreshed if they're close to
 ; expiring, or if any parameters related to extra IPs or domains in the cert
 ; change.
-; tlsautorefresh=true
+; tlsautorefresh=false
 
 ; The duration from generating the self signed certificate to the certificate
 ; expiry date. Valid time units are {s, m, h}.
 ; The below value is about 14 months (14 * 30 * 24 = 10080)
 ; tlscertduration=10080h
 
+
```

Comment:
nit: remove extra newline to keep the same size of gap between each option?

Reply:
removed